AGENTS.md

```
@@ -720,23 +694,42 @@ Bullets (optional, keep short):
 
 - Config: `.changeset/config.json` (baseBranch: `main`, changelog: GitHub)
 - Workspaces: `apps/*`, `packages/*` (pnpm)
 - Pending changesets: `.changeset/*.md`
 - CI workflow: `.github/workflows/release.yml`
 
-### Audible task completion (agent memory)
+# `cn()` usage
+
+You should never ever use the cn function unless you're defining a custom string or interpolating some dynamic data in a class name property. Stuff like
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)